openchain-blockchain-coinprism/azuredeploy.parameters.json

```diff
@@ -9,7 +9,7 @@
             "value": "Standard_A0"
         },
         "openchainVersion": {
-            "value": "0.5.0"
+            "value": "0.6.2"
```

Comment:
I'd need to add it back next time when two versions will be supported. Is there any harm in having it here?